localstack/services/stepfunctions/asl/component/common/catch/catcher_decl.py

```diff
@@ -74,8 +74,9 @@ def _extract_error_cause(failure_event: FailureEvent) -> dict:
                 f"Internal Error: invalid event details declaration in FailureEvent: '{failure_event}'."
             )
         spec_event_details: dict = list(failure_event.event_details.values())[0]
-        error = spec_event_details["error"]
-        cause = spec_event_details.get("cause") or ""
+        # If no cause or error fields are given, AWS binds an empty string; otherwise it attaches the value.
+        error = str() if "error" not in spec_event_details else spec_event_details["error"]
+        cause = str() if "cause" not in spec_event_details else spec_event_details["cause"]
```

Comment:
Is there a particular reason you've used `str()` over a simple empty string literal `""`? If not I'd argue the latter is a bit more clearer to the reader and should also avoid an unnecessary function call

Reply:
An alternative would be for both just doing a `.get("...", "")` which is a bit more "pythonic" and therefore clearer in purpose